src/app/manage-users/manage-user-creation/manage-user-creation.component.html

```diff
@@ -0,0 +1,3 @@
+<app-main-form 
+    (valuesSubmitted)="createNewUser($event)">
```

Comment:
Can we make this `valuesSubmitted` renamed to `onSubmit` as that is an event generally occurs when you actually submit a form

Reply:
renamed it to OnSubmit